docs_src/design/ucr/0003-Extending-Device-Data.md

```diff
@@ -0,0 +1,106 @@
+## Extending Device Data
+This UCR describes the Use Case for Application Services Extending Device Data for a given south-bound Device.
+
+### Submitters
+- Tom Brennan (Eaton)
+
+## Change Log
+- [pending](https://github.com/edgexfoundry/edgex-docs/pull/800) (2022-07-25)
+
+
+### Market Segments
+Any that deploy EdgeX systems with analytic, utility, or north-bound microservices that add new Device Resources
+that are extensions of the original south-bound or service-based Device data.
+
+### Motivation
+We find a consistent need as we design microservices for our industrial products:
+The new analytic, utility, and north-bound microservices almost always need to add Device Resources to manage their configuration, transforms, and status reporting. These Resources are usually needed on a per-Device basis (rather than just overall service configuration or status), which can be seen as extending (adding to) the data of the original south-bound devices.
```

Comment:
"These Resources are usually needed per device" instance or device class? i.e temp sensor 1 or all temp sensors from profile X?

Reply:
I was taking the simple view, "per device instance". But you raise a good point; having per device class would reduce the amount of data to be managed, at the cost of increasing the complexity of linking all the devices that were covered. 
And knowing our internal user-base, someone is going to insist on per-device instance even if we made per-device class the default.
Probably the internal reality (internal device class instances) will reduce it to per-device instance anyway;
and the default-value field in the device profiles lets you sort-of have it both ways.

Reply:
OK, please update to explicitly specify device instance.

Reply:
Done